specs/protocol/superchain-upgrades.md

```diff
@@ -256,7 +256,7 @@ but the matching L1-origin information may not be present at the time of activat
   [Governance Proposal](https://gov.optimism.io/t/upgrade-proposal-5-ecotone-network-upgrade/7669).
 - `v7.0.0`: 2024 Jul 10th - Fjord network upgrade (RIP-7212 precompile + FastLZ cost fn + Brotli compression).
   [Governance Proposal](https://gov.optimism.io/t/upgrade-proposal-9-fjord-network-upgrade/8236).
-- `v8.0.0`: 2024 Aug 19th (optimistic[^1]) - Granite network upgrade (Limit ecpairing input size + Reduce Channel Timeout).
+- `v8.0.0`: 2024 Aug 19th - Granite network upgrade (Limit ecpairing input size + Reduce Channel Timeout).
```

Comment:
I'm unclear why this is Aug 19th....

Reply:
You're right. It was [August 16 for Sepolia](https://sepolia.etherscan.io/address/0xfd7E6Ef1f6c9e4cC34F54065Bf8496cE41A4e2e8) and will be today for mainnet. I'll fix.

Reply:
I mean tomorrow. Fixed.